js/main.js

```diff
@@ -24,6 +24,97 @@ function saveNewEntry(event) {
   data.entries.unshift(entryInputs);
   $newEntryForm.reset();
   $urlImage.setAttribute('src', 'images/placeholder-image-square.jpg');
+  createEntry(data.entries[0]);
+  viewEntries();
 }
 
 $newEntryForm.addEventListener('submit', saveNewEntry);
+
+var $ulEntries = document.querySelector('.entries-list');
+
+function createEntry(entry) {
+  var liJournalEntry = document.createElement('li');
+  liJournalEntry.setAttribute('class', 'journal-entry');
+
+  var divRow = document.createElement('div');
+  divRow.setAttribute('class', 'row');
+  liJournalEntry.appendChild(divRow);
+
+  var divColumnHalf1 = document.createElement('div');
+  divColumnHalf1.setAttribute('class', 'column-half');
+  divRow.appendChild(divColumnHalf1);
+
+  var divEntryImageContainer = document.createElement('div');
+  divEntryImageContainer.setAttribute('class', 'entry-image-container');
+  divColumnHalf1.appendChild(divEntryImageContainer);
+
+  var imgEntryImage = document.createElement('img');
+  imgEntryImage.setAttribute('class', 'entry-image');
+  imgEntryImage.setAttribute('src', entry.photo);
+  imgEntryImage.setAttribute('alt', 'image for entry in code journal');
+  divEntryImageContainer.appendChild(imgEntryImage);
+
+  var divColumnHalf2 = document.createElement('div');
+  divColumnHalf2.setAttribute('class', 'column-half');
+  divRow.appendChild(divColumnHalf2);
+
+  var divEntryTextContainer = document.createElement('div');
+  divEntryTextContainer.setAttribute('class', 'entry-text-container');
+  divColumnHalf2.appendChild(divEntryTextContainer);
+
+  var divEntryTitle = document.createElement('div');
+  divEntryTitle.setAttribute('class', 'entry-title');
+  divEntryTextContainer.appendChild(divEntryTitle);
+
+  var headingThree = document.createElement('h3');
+  var entryHeading = document.createTextNode(entry.title);
+  headingThree.appendChild(entryHeading);
+  divEntryTitle.appendChild(headingThree);
+
+  var divEntryParagraph = document.createElement('div');
+  divEntryParagraph.setAttribute('class', 'entry-paragraph');
+  divEntryTextContainer.appendChild(divEntryParagraph);
+
+  var paragraphEntry = document.createElement('p');
+  var paragraphText = document.createTextNode(entry.note);
+  paragraphEntry.appendChild(paragraphText);
+  divEntryParagraph.appendChild(paragraphEntry);
+
+  $ulEntries.prepend(liJournalEntry);
```

Comment:
You should not be appending in this function, you should be returning the `li` that holds the DOM tree for the entry